src/it/scala/org/renci/cam/it/QueryServiceTest.scala

```diff
@@ -321,7 +321,7 @@ object QueryServiceTest extends DefaultRunnableSpec {
 
   val testDILIGeneList = suite("testDILIGeneList")(
     testM("DILIGeneList") {
-      val diliGeneList = Files.readAllLines(Paths.get("src/it/resources/dili-gene-list-alt.txt")).asScala.map(a => IRI(a.trim())).toList
+      val diliGeneList = Files.readAllLines(Paths.get("src/it/resources/dili-gene-list.txt")).asScala.map(a => IRI(a.trim())).toList
```

Comment:
@jdr0887 Hi Jason! Jim wanted to me to check with you about why the filename here doesn't correspond to the name of the file as checked into GitHub. Do you remember why these are different? The test appears to pass if I remove the `-alt` from the filename, so I'm guessing that the filenames just got out of sync at some point, but I wanted to make sure the alt file wasn't a different file from the main one. Thanks!